svir/test/data/hazard/hazard_curves_SA(0.1).csv

```diff
@@ -1,3 +1,3 @@
 lon,lat,mean_SA(0.1)_0.005,mean_SA(0.1)_0.007,mean_SA(0.1)_0.0098,mean_SA(0.1)_0.0137,mean_SA(0.1)_0.0192,mean_SA(0.1)_0.0269,mean_SA(0.1)_0.0376,mean_SA(0.1)_0.0527,mean_SA(0.1)_0.0738,mean_SA(0.1)_0.103,mean_SA(0.1)_0.145,mean_SA(0.1)_0.203,mean_SA(0.1)_0.284,mean_SA(0.1)_0.397,mean_SA(0.1)_0.556,mean_SA(0.1)_0.778,mean_SA(0.1)_1.09,mean_SA(0.1)_1.52,mean_SA(0.1)_2.13,quantile-0.05_SA(0.1)_0.005,quantile-0.05_SA(0.1)_0.007,quantile-0.05_SA(0.1)_0.0098,quantile-0.05_SA(0.1)_0.0137,quantile-0.05_SA(0.1)_0.0192,quantile-0.05_SA(0.1)_0.0269,quantile-0.05_SA(0.1)_0.0376,quantile-0.05_SA(0.1)_0.0527,quantile-0.05_SA(0.1)_0.0738,quantile-0.05_SA(0.1)_0.103,quantile-0.05_SA(0.1)_0.145,quantile-0.05_SA(0.1)_0.203,quantile-0.05_SA(0.1)_0.284,quantile-0.05_SA(0.1)_0.397,quantile-0.05_SA(0.1)_0.556,quantile-0.05_SA(0.1)_0.778,quantile-0.05_SA(0.1)_1.09,quantile-0.05_SA(0.1)_1.52,quantile-0.05_SA(0.1)_2.13,quantile-0.5_SA(0.1)_0.005,quantile-0.5_SA(0.1)_0.007,quantile-0.5_SA(0.1)_0.0098,quantile-0.5_SA(0.1)_0.0137,quantile-0.5_SA(0.1)_0.0192,quantile-0.5_SA(0.1)_0.0269,quantile-0.5_SA(0.1)_0.0376,quantile-0.5_SA(0.1)_0.0527,quantile-0.5_SA(0.1)_0.0738,quantile-0.5_SA(0.1)_0.103,quantile-0.5_SA(0.1)_0.145,quantile-0.5_SA(0.1)_0.203,quantile-0.5_SA(0.1)_0.284,quantile-0.5_SA(0.1)_0.397,quantile-0.5_SA(0.1)_0.556,quantile-0.5_SA(0.1)_0.778,quantile-0.5_SA(0.1)_1.09,quantile-0.5_SA(0.1)_1.52,quantile-0.5_SA(0.1)_2.13,quantile-0.95_SA(0.1)_0.005,quantile-0.95_SA(0.1)_0.007,quantile-0.95_SA(0.1)_0.0098,quantile-0.95_SA(0.1)_0.0137,quantile-0.95_SA(0.1)_0.0192,quantile-0.95_SA(0.1)_0.0269,quantile-0.95_SA(0.1)_0.0376,quantile-0.95_SA(0.1)_0.0527,quantile-0.95_SA(0.1)_0.0738,quantile-0.95_SA(0.1)_0.103,quantile-0.95_SA(0.1)_0.145,quantile-0.95_SA(0.1)_0.203,quantile-0.95_SA(0.1)_0.284,quantile-0.95_SA(0.1)_0.397,quantile-0.95_SA(0.1)_0.556,quantile-0.95_SA(0.1)_0.778,quantile-0.95_SA(0.1)_1.09,quantile-0.95_SA(0.1)_1.52,quantile-0.95_SA(0.1)_2.13
```

Comment:
The problem with the CSV tests is they are very fragile. On different platforms the numbers could be slightly different. With different versions of shapely/libgeos/numpy/etc the numbers could be slightly different. The parameters of the demos could change in the future and the numbers (even the number of rows and columns) could change. I suggest to simply check that a file is generated with at least two rows and three columns.

Reply:
Done